Updating the Automatic Run Triggering and VCS Branch settings docs to… (#193)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/vcs.mdx

| **Version contains a prefix** | `\d+.\d+.\d+$`            | `\\d+.\\d+.\\d+$`        |
| **Version contains a suffix** | `^\d+.\d+.\d+`            | `^\\d+.\\d+.\\d+`        |

> Runs will be triggered on all tags matching this pattern, regardless of the value in the [VCS Branch](#vcs-branch) setting

Reviewer: ```suggestion
Terraform Cloud triggers runs for all tags matching this pattern, regardless of the value in the [VCS Branch](#vcs-branch) setting.
```

### VCS Branch

> This designates which branch of the repository Terraform Cloud should use in the workspace if you are using [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If left blank, Terraform Cloud uses the repository's default branch. If you are triggering runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of what this field is set to.

Reviewer: ```suggestion
This setting designates which branch of the repository Terraform Cloud should use when the workspace is set to [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If you leave this setting blank, Terraform Cloud uses the repository's default branch. If the workspace is set to trigger runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of the branch specified in this setting.
```

> This setting designates which branch of the repository HCP Terraform should use when the workspace is set to [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If you leave this setting blank, Terraform Cloud uses the repository's default branch. If the workspace is set to trigger runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of the branch specified in this setting.

Reviewer 2: ```suggestion
This setting designates which branch of the repository HCP Terraform should use when the workspace is set to [Always Trigger Runs](#always-trigger-runs) or [Only trigger runs when files in specified paths change](#only-trigger-runs-when-files-in-specified-paths-change). If you leave this setting blank, HCP Terraform uses the repository's default branch. If the workspace is set to trigger runs when a [git tag is published](#trigger-runs-when-a-git-tag-is-published), all tags will trigger runs, regardless of the branch specified in this setting.
```
